plugins/modules/zos_fetch.py

```diff
@@ -565,6 +574,7 @@ def run_module():
             sftp_port=dict(type="int", required=False),
             ignore_sftp_stderr=dict(type="bool", default=False, required=False),
             local_charset=dict(type="str"),
+            tmphlq=dict(required=False, type="str", default=None),
```

Comment:
There is confusion... what I thought I saw was 'tmphlq' is a string/parameter, and tmp_hlq is a dictionary. As I look more closely, I'm seeing them intermixed. I'll try to put comment notes where I see tmphlq as a dictionary.

Reply:
Resolving because I created issue https://github.com/ansible-collections/ibm_zos_core/issues/387 - it was a good comment and point, please review the issue as well @fernandofloresg , also see the other work item that came from this PRs review https://github.com/ansible-collections/ibm_zos_core/issues/386